crates/cli/cli/src/commands/client/create.rs

```diff
@@ -0,0 +1,149 @@
+    #[clap(
+        long = "reference-chain",
+        required = true,
+        value_name = "REFERENCE_CHAIN_ID",
+        help_heading = "REQUIRED"
+    )]
+    reference_chain_id: ChainId,
+
+    /// The maximum allowed clock drift for this client.
+    ///
+    /// The clock drift is a correction parameter. It helps deal with clocks
+    /// that are only approximately synchronized between the source and destination chains
+    /// of this client.
+    /// The destination chain for this client uses the clock drift parameter when deciding
+    /// to accept or reject a new header (originating from the source chain) for this client.
+    /// If this option is not specified, a suitable clock drift value is derived from the chain
+    /// configurations.
+    #[clap(long = "clock-drift", value_name = "CLOCK_DRIFT")]
+    clock_drift: Option<humantime::Duration>,
+
+    /// Override the trusting period specified in the config.
+    ///
+    /// The trusting period specifies how long a validator set is trusted for
+    /// (must be shorter than the chain's unbonding period).
+    #[clap(long = "trusting-period", value_name = "TRUSTING_PERIOD")]
+    trusting_period: Option<humantime::Duration>,
+
+    /// Override the trust threshold specified in the configuration.
+    ///
+    /// The trust threshold defines what fraction of the total voting power of a known
+    /// and trusted validator set is sufficient for a commit to be accepted going forward.
+    #[clap(
+        long = "trust-threshold",
+        value_name = "TRUST_THRESHOLD",
+        value_parser = parse_trust_threshold
+    )]
+    trust_threshold: Option<TrustThreshold>,
+}
+
+impl Runnable for ClientCreate {
+    async fn run(&self, builder: CosmosBuilder) -> Result<()> {
+        let host_chain_config =
+            builder
+                .config
+                .find_chain(&self.host_chain_id)
+                .ok_or_else(|| {
+                    eyre!(
+                        "No chain configuration found for chain `{}`",
+                        self.host_chain_id
+                    )
+                })?;
+
+        let reference_chain_config = builder
+            .config
+            .find_chain(&self.reference_chain_id)
+            .ok_or_else(|| {
+                eyre!(
+                    "No chain configuration found for chain `{}`",
+                    self.reference_chain_id
+                )
+            })?;
+
+        let host_chain = builder.build_chain(&self.host_chain_id).await?;
+        let reference_chain = builder.build_chain(&self.reference_chain_id).await?;
+
+        let options = CreateOptions {
+            max_clock_drift: self.clock_drift.map(|d| d.into()),
+            trusting_period: self.trusting_period.map(|d| d.into()),
+            trust_threshold: self.trust_threshold,
+        };
+
+        let settings =
+            ClientSettings::for_create_command(options, host_chain_config, reference_chain_config);
```

Comment:
We should later find a way to create the client settings without having to supply the whole config of two chains.